recipes/librhash/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import CMake, ConanFile, tools
+
+
+class LibRHashConan(ConanFile):
+    name = "librhash"
+    description = "Great utility for computing hash sums"
+    topics = ("conan", "rhash", "hash", "checksum")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://rhash.sourceforge.net/"
+    license = "MIT"
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake",
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_openssl": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_openssl": False,}
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
```

Comment:
btw, gettext it's an optional requirement on Linux

Reply:
ping @madebr

Reply:
gettext is currently not packaged by cci, I guess it's better to remove it?

Reply:
gettext is only needed to regenerate the translated files when some strings are changed.
This is useful during development. So I think gettext could be safely removed.